apps/server/src/modules/user/uc/dto/user.dto.ts

```diff
@@ -23,13 +23,13 @@ export class UserDto {
 
 	lastName: string;
 
-	roleIds: string[] = [];
+	roleIds: EntityId[] = [];
 
 	schoolId: string;
 
 	ldapDn?: string;
 
-	ldapId?: string;
+	externalId?: string;
```

Comment:
Please create jsdoc for this field. Renamed ldapId to externalId because....

Reply:
done